wsgi_kerberos.py

```diff
@@ -182,9 +182,17 @@ def __call__(self, environ, start_response):
         if authorization is None:
             return self._unauthorized(environ, start_response)
 
-        # If we have an 'Authorization' header, extract the client's token and
-        # attempt to authenticate with it.
-        client_token = ''.join(authorization.split()[1:])
+        # We have an 'Authorization' header -> should start with 'Negotiate '.
+        parsed = authorization.split(None, 1)
+        if len(parsed) < 2 or parsed[0].lower() != 'negotiate':
+            return self._unauthorized(
+                environ,
+                start_response,
+                # If extra_body is added from PR #10, use it to explain what went wrong:
+                # extra_body=b': Expected "Authorization: Negotiate <token>"',
```

Comment:
Remove this

Reply:
Done in the latest revision. Another look?

Reply:
Note, I updated #18 to incorporate this change. If this PR now looks good to merge, and #18 does too, you can just merge #18 and it should still automatically resolve this PR (#17) as well as issue #16.